### PR TITLE
Turns URLs starting with 'www.' into links

### PR DESCRIPTION
**Pull request by rubenswieringa**
urlEmbed() will now also look for URLs starting with _“www.”_ (not just _“http://”_)

Additional stuff:
- I removed the `((href|src)=["']|)` portion of the regex, since I could see no obvious use for this.
- I also removed the braces around the body of the regex (since this is a simple string-replace we're not actually using that capturing group)

### CODE REVIEW

File: src/ng-embed.js [hunk 1]
```diff
@@ -133,10 +133,16 @@
 
                 function urlEmbed(str) {
 
-                    var urlRegex = /((href|src)=["']|)(\b(https?|ftp|file):\/\/[-A-Z0-9+()&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])/ig;
+                    var urlRegex = /\b(?:(https?|ftp|file):\/\/|www\.)[-A-Z0-9+()&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|]/ig;
```
Comment (rubenswieringa) [inline src/ng-embed.js]: better diff:
screen shot 2015-08-17 at 11 48 22

File: src/ng-embed.js [hunk 2]
```diff
@@ -133,10 +133,16 @@
...
+                    var protocolRegex = /^[a-z]+\:\/\//i;
 
-                    var strReplaced = str.replace(urlRegex, function (match) {
-                            return '<a href="' + match + '" target="' + options.linkTarget + '">' + match + '</a>';
+                    var strReplaced = str.replace(urlRegex, function (text) {
+                            if (protocolRegex.test(text))
+                                var url = text;
```
Comment (ritz078) [inline src/ng-embed.js]: Always use curly braces as it won't pass jshint while build.

**Comment by ritz078** [comment]
You should always run `grunt`and create a build version before opening a PR.

**Comment by rubenswieringa** [comment]
@ritz078 ok, grunt-task is run and curly brackets fixed.

Also, figured this might come in handy for next time someone wants to contribute:
» [**Tell contributors to run grunt-task before filing pull-requests**](https://github.com/ritz078/ngEmbed/pull/9)

**Comment by rubenswieringa** [comment]
@ritz078 do you need me to make any further changes?

**Comment by rubenswieringa** [comment]
@ritz078 is there anything in the way of merging this PR? Anything I can help out with? I'd love for this functionality to be in the package :)

**Comment by ritz078** [comment]
Hi @rubenswieringa , I will test it and merge today if everything is fine. I am just a busy these days with [embed.js](https://github.com/ritz078/embed.js) as i am converting it into a pure JS plugin with ES6 code. Thanks.

**Comment by rubenswieringa** [comment]
Sweet! Thanks!

**Comment by rubenswieringa** [comment]
@ritz078 could you also make a new release on master?

**Pull request merged**